src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/AttachExceptionRecordToExistingCaseTest.java

```diff
@@ -130,6 +133,57 @@ public void should_attach_exception_record_to_the_existing_case_with_evidence_do
         verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
     }
 
+    @Test
+    public void should_attach_supplementary_evidence_with_pending_payments() throws Exception {
+        //given
+        CaseDetails caseDetails = ccdCaseCreator.createCase(emptyList(), now()); // with no scanned documents
+        CaseDetails exceptionRecord = createExceptionRecord(
+            "envelopes/supplementary-evidence-envelope-with-payment.json"
+        );
+
+        // when
+        attachExceptionRecord(caseDetails, exceptionRecord, null);
+
+        //then
+        await("Exception record with pending payments is attached to the case")
+            .atMost(60, TimeUnit.SECONDS)
+            .pollDelay(2, TimeUnit.SECONDS)
+            .until(() -> isExceptionRecordAttachedToTheCase(caseDetails, 1));
+
+        verifyExistingCaseIsUpdatedWithExceptionRecordData(caseDetails, exceptionRecord, 1);
```

Comment:
Shouldn't there be a warning instead? I can't see any ignoreWarnings flag passed to the request.

Reply:
I'll add `ignoreWarnings` flag with the request